.github/workflows/pr-backend-code-checks.yml

```diff
@@ -23,6 +23,16 @@ jobs:
         with:
           go-version: '1.23.7'
           cache: true
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y build-essential shared-mime-info
```

Comment:
these might not be necessary. Try removing this step and seeing if it still works?

Reply:
It does not; thanks!